stepcvt/cli/task.py

```diff
@@ -0,0 +1,59 @@
+import json
+from ..project import *
+from stepcvt import project
+
+
+def stlconvert(args):
+    # check for jsonfile
+    if args.jsonfile is None:
+        print("ERROR: Need to provide a jsonfile")
+        return 1
+
+    # try to get info from jsonfile
+    try:
+        with open(args.jsonfile, "r") as jf:
+            d = json.load(jf)
+            p = Project.from_dict(d)
+    except FileNotFoundError as fe:
+        print(f"ERROR: {args.jsonfile} doesn't exist")
+        return 1
+    except json.JSONDecodeError:
+        print("ERROR: Invalid json syntax")
+        return 1
+
+    if not p.sources:
+        print("ERROR: No existing source, add one first")
+    else:
+        source = p.sources[0]
+
+    if args.partID:
+        partID = args.partID
+    else:
+        print("ERROR: Need to specify at least one partid if --all is not included")
+        return 1
+
+    if args.rotation == True:
+        rotation = args.rotation
+    if args.linearTolerance == True:
+        linearTol = args.linearTolerance
+    else:
+        linearTol = 0.1
+    if args.angularTolerance == True:
+        angularTol = args.angularTolerance
+    else:
+        angularTol = 0.1
+
+    partID = project.STLConversionInfo(
+        rotation=rotation, linearTolerance=linearTol, angularTolerance=angularTol
+    )
+    partID.rotate(source)
+
+    with open(args.jsonfile, "w") as jf:
+        json.dump(partID.to_dict(), jf)
```

Comment:
Should we update the jsonfile with the changed `project`? Right now the jsonfile only contains the information of a `STLConversionInfo`.